packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.tsx

```diff
@@ -1,210 +1,10 @@
-import * as React from 'react';
-import { BaseComponent, DelayedRender, css, getId } from '../../Utilities';
-import { IconButton } from '../../Button';
-import { Icon } from '../../Icon';
-import { IMessageBarProps, MessageBarType } from './MessageBar.types';
-import * as stylesImport from './MessageBar.scss';
-const styles: any = stylesImport;
-
-export interface IMessageBarState {
-  labelId?: string;
-  showContent?: boolean;
-  expandSingleLine?: boolean;
-}
-
-export class MessageBar extends BaseComponent<IMessageBarProps, IMessageBarState> {
-  public static defaultProps: IMessageBarProps = {
-    messageBarType: MessageBarType.info,
-    onDismiss: undefined,
-    isMultiline: true
-  };
-
-  private ICON_MAP = {
-    [MessageBarType.info]: 'Info',
-    [MessageBarType.warning]: 'Info',
-    [MessageBarType.error]: 'ErrorBadge',
-    [MessageBarType.blocked]: 'Blocked2',
-    [MessageBarType.remove]: 'Blocked', // TODO remove deprecated value at >= 1.0.0
-    [MessageBarType.severeWarning]: 'Warning',
-    [MessageBarType.success]: 'Completed'
-  };
-
-  constructor(props: IMessageBarProps) {
-    super(props);
-
-    this.state = {
-      labelId: getId('MessageBar'),
-      showContent: false,
-      expandSingleLine: false
-    };
-  }
-
-  public render(): JSX.Element {
-    const { isMultiline } = this.props;
-
-    return isMultiline ? this._renderMultiLine() : this._renderSingleLine();
-  }
-
-  private _getActionsDiv(): JSX.Element | null {
-    if (this.props.actions) {
-      return (
-        <div
-          className={
-            this.props.isMultiline
-              ? 'ms-MessageBar-actions ' + styles.actions
-              : 'ms-MessageBar-actionsSingleLine ' + styles.actionsSingleLine
-          }
-        >
-          {this.props.actions}
-        </div>
-      );
-    }
-    return null;
-  }
-
-  private _getClassName(): string {
-    return css(this.props.className, 'ms-MessageBar', styles.root, {
-      ['ms-MessageBar ' + styles.root]: this.props.messageBarType === MessageBarType.info,
-      ['ms-MessageBar--error ' + styles.rootIsError]: this.props.messageBarType === MessageBarType.error,
-      ['ms-MessageBar--blocked ' + styles.rootIsBlocked]:
-        this.props.messageBarType === MessageBarType.blocked || this.props.messageBarType === MessageBarType.remove, // TODO remove deprecated value at >= 1.0.0
-      ['ms-MessageBar--severeWarning ' + styles.rootIsSevereWarning]:
-        this.props.messageBarType === MessageBarType.severeWarning,
-      ['ms-MessageBar--success ' + styles.rootIsSuccess]: this.props.messageBarType === MessageBarType.success,
-      ['ms-MessageBar--warning ' + styles.rootIsWarning]: this.props.messageBarType === MessageBarType.warning
```

Comment:
Hmm should these global names have been preserved? As I was looking through Pivot I was wondering if I should preserve these myself... these were in the snapshot output previosuly too, but are no longer